cmd/git/git.go

```diff
@@ -17,6 +18,7 @@ import (
 	"time"
 
 	"github.com/LF-Engineering/insights-datasource-git/gen/models"
```

Comment:
I *think* you should also remove `"github.com/LF-Engineering/insights-datasource-git/gen/models"` and all swagger-related files: swagger directory, JSONs, references, Makefile stuff, etc.

Reply:
removed.